test/utils.py

```diff
@@ -76,7 +76,8 @@ def cibuildwheel_run(project_path, package_dir='.', env=None, add_env=None, outp
 
 
 def expected_wheels(package_name, package_version, manylinux_versions=None,
-                    macosx_deployment_target='10.9', machine_arch=None):
+                    macosx_deployment_target='10.9', machine_arch=None, *,
+                    filter_27=IS_WINDOWS_RUNNING_ON_TRAVIS):
```

Comment:
I don't love this signature growing but I can't see a way around it either! Nitpick, but `filter_27` might be a little ambiguous, maybe `exclude_27` would be clearer?

Reply:
Is there a reason to go for kwargs-only arguments? This might be the only place in the whole of cibuildwheel, currently?

Reply:
No, there are some in other places (build selectors, for example). They force you to remain readable. I'd like to move the star farther to the left; though `"10.9" or `"x86_64"` might be guessable on it's own, it's much clearer with the keyword. And a random True at the end would be a disaster!

Reply:
Ehh, maybe it's 50/50. I didn't write the ones in logger or c; the main reason not to have keyword only arguments is for compatibility with Python 2. They should always be used if it's not obvious from the context or type of argument.

Reply:
Oh, there is more. Eh, not worth changing, then. :-)

Reply:
(It's just ... internal API; I don't care too much about that; it's mostly the calling code that should be careful, then. But it's not _wrong_ either, ofc)